torch/_inductor/codegen/common.py

```diff
@@ -450,6 +450,39 @@ def remainder(a, b):
     def load_seed(name, offset):
         return ops.load(name, sympy.Integer(offset))
 
+class DeviceOverrides():
```

Comment:
How about changing it to `WrapperOpsOverrides`

Reply:
> How about changing it to `WrapperOpsOverrides`

The interface also used by TritonScheduling, is it OK to change to  `WrapperOpsOverrides`?

Reply:
Make sense. Then how about changing it to `DeviceOpsOverrides`?